### PR TITLE
fmt.Errorf replaces errors

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
**What this PR does / why we need it**:

fmt.Errorf replaces errors
importing errors is redundant.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by spiffxp** [comment]
@k8s-bot ok to test

File: pkg/kubectl/rolling_updater.go [hunk 1]
```diff
@@ -616,11 +615,11 @@ func CreateNewControllerFromCurrentController(rcClient coreclient.ReplicationCon
 	}
 
 	if len(newRc.Spec.Template.Spec.Containers) > 1 && len(cfg.Container) == 0 {
-		return nil, goerrors.New("Must specify container to update when updating a multi-container pod")
+		return nil, fmt.Errorf("Must specify container to update when updating a multi-container pod")
```
Comment (kargakis) [inline pkg/kubectl/rolling_updater.go]: Can you also lowercase both of these errors? By convention, go errors should start lowercased.

**Comment by zhangxiaoyu-zidif** [comment]
Hi, @kargakis I lowercase the errors. Thanks for reviewing.
PTAL

**Comment by zhangxiaoyu-zidif** [comment]
Hi, @brendandburns, could you approve it. Thanks very much.

**Comment by zhangxiaoyu-zidif** [comment]
@brendandburns 
PTAL

**Comment by zhangxiaoyu-zidif** [comment]
ping @brendandburns

**Comment by zhangxiaoyu-zidif** [comment]
@k8s-bot unit test this

**Pull request merged**